models/products-model.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const productsData = require("./products-data");
+
+const productSchema = new Schema({
+  id: Schema.Types.ObjectId,
+  name: String,
+  image: String,
+  price: Number,
+  inStock: Boolean,
+  fastDelivery: Boolean,
+  rating: Number,
+  discount: Number,
+  category: String,
+  rating:Number,
+  quantity:Number
+});
+
+const Product = mongoose.model("Product", productSchema);
+
+// one time add in db
+async function fillProductsCollection() {
```

Comment:
Its better you use ES6 syntax everywhere. Like 

const fillProductsCollection = () => { 
}